SPL documentation for InputLookup command (#118)
siglens/siglens-docs: docs/spl-docs/inputlookup-command.md

## Description
The `inputlookup` command is used to search the contents of a CSV-based lookup table. This command allows you to retrieve and filter data stored in a lookup table.

## Syntax

Required syntax is in **bold**.

| **inputlookup**\
[append=\<boolean\>]\
[start=\<boolean\>]\
[max=\<boolean\>]\
**\<filename\>**
[WHERE \<search-query\>]

### Required Arguments

#### \<filename\>
**Syntax:** `<string>`\
**Description:**\
The name of the lookup file. It must end with either `.csv` or `.csv.gz`.

### Optional Arguments

#### append
**Syntax:** `append=<boolean>`\
**Description:**\
If set to `true`, the data from the lookup file is appended to the current results instead of replacing them.\
This option must be set to `true` if the `inputlookup` command is not the first command in the search.\
**Default:** `false`

#### start
**Syntax:** `start=<int>`\
**Description:**\
Specify the 0-based offset for the first event to read. For example, if `start=0`, it begins with the first event. If `start=5`, it begins with the sixth event.\
**Default:** 0

#### max
**Syntax:** `max=<int>`\
**Description:**\
Indicate the maximum number of events to read from the file.\

> **Default:** 1000000000

Reviewer: nit: it's a little hard to read all the zeros
```suggestion
**Default:** 1000000000 (1 billion)
```

Author: addressed in the latest commit.

#### WHERE \<search-query\>
**Syntax:** `<boolean-expression>`\
**Description:**
Use this clause to enhance search performance by pre-filtering data from the lookup table.\
It supports a limited set of comparison operators: `=`, `!=`, `<`, `>`, `<=`, `>=`, `AND`, `OR`, `NOT`.\
Any combination of these operators is allowed. The `*` character can be used to perform wildcard string searches.

> 

Reviewer: It would be good to mention on `where` and `max` interact. E.g., if `max=10` and `where` is set, will it get the first 10 and then filter out those that don't match the `where`, or will it filter out the records and then get the first 10 that still remain?

Author: addressed in the latest commit.

## Usage

> The `inputlookup` command is an event-generating command. This command uses a leading pipe character and should be the first command in a search. It can also be used later in the search, but `append` must be set to `true`.
> 

Reviewer: nit: this is a bit misleading:
>  This command uses a leading pipe character and should be the first command in a search. It can also be used later in the search, but `append` must be set to `true`.

```suggestion
The `inputlookup` command is an event-generating command. When used as the first command in the query, a preceding pipe is required. If it's not the first command, it must have `append=true`.
```

You could also mention that it's generally the first command in the query (I'm don't know if that's true), if that will be useful to users

Author: addressed in the latest commit.

Users can upload local `csv` and `csv.gz` files by navigating to Lookup Tab > Add New Lookup File. All the lookup files will be stored under the `data/lookups` directory. These files can be used for further queries.

## Examples

The following example reads the data.csv lookup file.

```
| inputlookup data.csv
```

## Use-Case Examples

> ### 43. Analyzing Network Traffic Actions Based on Subnet

Reviewer: Great example
